wiki/Beatmap_Editor/Beat_Snap_Divisor/de.md

```diff
@@ -1,20 +1,60 @@
+Beat Snap Divisor (Deutsch)
+==============================
+
 osu!, Taiko und Catch the Beat
 ------------------------------
 
 Der Beat Snap Divisor wird benutzt, um die Positionen, auf denen Noten gelegt werden können, zu limitieren. Um die Zählzeit zu erhöhen, bewegen Sie einfach den Balken nach rechts.
 
-Jede dieser Einstellungen benutzt für die Ticks in der [Timeline](DE:Timelines "wikilink") unterschiedliche Farben. Die Ticks sind wie folgt:
+Jede dieser Einstellungen benutzt für die Ticks in der [Timeline](/wiki/Beatmap_Editor/Timelines) unterschiedliche Farben. Die Ticks sind wie folgt:
+
+### Divisor 1/1
+
+![](/wiki/shared/BSD_1_1b.jpg)
+
+- Volle-Beats
+- Nur weiße Ticks
+- Am besten für Easy Schwierigkeitsstufen geeignet
+
+### Divisor 1/2
+
+![](/wiki/shared/BSD_1_2.jpg)
+
+- Halbe-Beats
+- Fügt rote Ticks hinzu
+
+### Divisor 1/3
+
+![](/wiki/shared/BSD_1_3.jpg)
+
+- Tripel*
+- Hat zwei lila Ticks zwischen jeden weißen Tick
+- Compound timing
+
+### Divisor 1/4
 
-| Grundschlag                      | Divisor | Beat              | Tick Noten                                               | Noten                                            |
-|----------------------------------|---------|-------------------|----------------------------------------------------------|--------------------------------------------------|
-| ![](BSD 1 1b.jpg "BSD 1 1b.jpg") | 1/1     | Volle-Beats       | Nur weiße Ticks                                          | Am besten für Easy Schwierigkeitsstufen geeignet |
-| ![](BSD 1 2.jpg "BSD 1 2.jpg")   | 1/2     | Halbe-Beats       | Fügt rote Ticks hinzu                                    |                                                  |
-| ![](BSD 1 3.jpg "BSD 1 3.jpg")   | 1/3     | Tripel\*          | Hat zwei lila Ticks zwischen jeden weißen Tick           | Compound timing                                  |
-| ![](BSD 1 4.jpg "BSD 1 4.jpg")   | 1/4     | Viertel-Beats     | Fügt blaue Ticks als Ergänzung zu roten Ticks hinzu      | Nützlich für generelles Mapping                  |
-| ![](BSD 1 6.jpg "BSD 1 6.jpg")   | 1/6     | Doppelte-Tripel\* | Hat zwei lila Ticks zwischen jeden roten Tick            | Compound timing                                  |
-| ![](BSD 1 8.jpg "BSD 1 8.jpg")   | 1/8     | Achtel-Beats      | Fügt gelbe Ticks als Ergänzung zu roten und blauen hinzu |                                                  |
+![](/wiki/shared/BSD_1_4.jpg)
 
-Halten Sie die **ALT** Taste gedrückt und drehen Sie am Mausrad, um den [Distance Snap](DE:Distance_Snap "wikilink") zu verändern.
+- Viertel-Beats
+- Fügt blaue Ticks als Ergänzung zu roten Ticks hinzu
+- Nützlich für generelles Mapping
+
+### Divisor 1/6
+
+![](/wiki/shared/BSD_1_6.jpg)
```

Comment:
L44: Add text on ``[]`` to phrase which supposed to be there if the image doesn't load